packages/content/templates/nuxt-content.js

```diff
@@ -79,6 +81,24 @@ function slotsToData (node, h, doc) {
   return data
 }
 
+function processDynamicNode (node, h, doc) {
+  const { props } = node
+  const key = Object.keys(props).find((key) => rxIs.test(key))
+  const value = props[key]
+
+  if (value) {
+    node.tag = value in doc ? doc[value] : evalInContext(value, doc)
+    node.props.tag = DYMAMIC_COMOPNENT
+    node.children = node.children.filter((child) => {
+      const _child = processNode(child, h, doc)
+      return !(typeof _child === 'string' && /^\s*$/.test(_child))
+    })
```

Comment:
If I used `component` component in Vue template, the parsed node's `children` doesn't include nodes whose type is `"text"` and value is just like `\n`.